.github/workflows/ci.yml

```diff
@@ -20,7 +19,6 @@ on:
     paths-ignore:
       - 'demo/**'
       - 'docker/**'
-      - 'Dockerfile'
```

Comment:
Can you change these lines back? I don't want the CI to run when only changes to the Dockerfile are made. When we update the Dockerfile, it's extremely unlikely that we'll update it to be out of date with the `stack.yaml`, so when we make a change to `stack.yaml` that is likely to move things out of sync, the lines you've added below to check the Dockerfile version will still run and notify us appropriately.

Reply:
sure

Reply:
Can you remove it here as well, please? For the same reason as before, I don't want to run CI when only the Dockerfile has changed since our CI doesn't build the Docker image.